src/main/java/uk/gov/hmcts/divorce/solicitor/service/SolicitorSubmitPetitionService.java

```diff
@@ -57,4 +60,22 @@ public OrderSummary getOrderSummary() {
             )
             .build();
     }
+
+    public ListValue<Payment> getDummyPayment(OrderSummary orderSummary) {
```

Comment:
it kind of feels that this thing should return a Payment and then the callback handler should add the ListValue wrapper. I know it's only temporary but in theory the Payment might not always be used in a list

Reply:
👍 - was in two minds about the return type for this one - I agree that returning a Payment makes the most sense